src/Addon/Helpers/Environment.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace GiveAddon\Addon\Helpers;
+
+/**
+ * Helper class responsible for checking the add-on environment.
+ *
+ * @package     GiveAddon\Addon\Helpers
+ * @copyright   Copyright (c) 2020, GiveWP
+ */
+class Environment {
+
+	/**
+	 * Check eneviroment.
+	 *
+	 * @return void
+	 */
+	public static function checkEnviroment() {
+		// Check is GiveWP active
+		if ( ! static::isGiveActive() ) {
+			add_action( 'admin_notices', [ Notices::class, 'giveInactive' ] );
```

Comment:
@ravinderk that if statement checks if GiveWP is active, and if it's not, it will register an action which will display the notice. Using Hooks helper here will trigger a fatal error.